ports/atmel-samd/common-hal/watchdog/WatchDogMode.c

```diff
@@ -0,0 +1,26 @@
+/*
+ * This file is part of the MicroPython project, http://micropython.org/
+ *
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2021 dearmash for Adafruit Industries
```

Comment:
You are not paid by Adafruit, so it can just be copyright by you.

Reply:
No need for my name, just leaving Adafruit.